torch/_inductor/utils.py

```diff
@@ -44,6 +44,20 @@
 import sympy
 
 import torch
+
+
+GPU_TYPES = ["cuda", "xpu"]
+
+
+def _get_gpu_type():
+    avail_gpus = [x for x in GPU_TYPES if getattr(torch, x).is_available()]
+    assert len(avail_gpus) <= 1
+    gpu_type = "cuda" if len(avail_gpus) == 0 else avail_gpus.pop()
+    return gpu_type
+
+
+GPU_TYPE = _get_gpu_type()
```

Comment:
Can you please move this code below the `from XYZ import ...` block? (may be towards the end of the file, unless other utils depend on it

Reply:
Thank you, your comments helps a lot. In this place, since I need to use _get_gpu_type() to generalize the device-bias code inside dynamo in my later PR, I need to import _get_gpu_type inside dynamo, and in order to avoid the loop of importing, I just put this function in this file before the import of dynamo. Do you have any suggestions for this scenario? Or do I need to put it outside the inductor?

Reply:
@malfet can you help review the changes?

Reply:
> Can you please move this code below the from XYZ import ... block? (may be towards the end of the file, unless other utils depend on it

Hi, @malfet , do you have any comment for this change?